docs/content/docs/data-structures/sequence.md

```diff
@@ -0,0 +1,198 @@
+determine the value.
+
+## SharedString
+
+The SharedString is a specialized data structure for handling collaborative text. It is based on a more general
+Sequence data structure but has additional features that make working with text easier.
+
+In addition to text, a SharedString can also contain markers. Markers can be used to store metadata at positions within
+the text, like the details of an image or Fluid object that should be rendered with the text.
+
+Both markers and text are stored as segments in the SharedString. Text segments will be split and merged when
+modifications are made to the SharedString and will therefore have variable length matching the length of the text
+content they contain. Marker segments are never split or merged, and always have a length of 1.
+
+The length of the SharedString will be the combined length of all the text and marker segments. Just like with other
+sequences, when talking about positions in a SharedString we use the terms near and far. The nearest position in a
+SharedString is 0, and the farthest position is its length. When comparing two positions the nearer positions is closer
+to 0, and the farther position is closer to the length.
+
+### Examples
+
+- Rich Text Editor Implementations
+  - [webflow](https://github.com/microsoft/FluidFramework/tree/main/examples/data-objects/webflow)
+  - [flowView](https://github.com/microsoft/FluidFramework/blob/main/examples/data-objects/client-ui-lib/src/controls/flowView.ts)
+
+- Integrations with Open Source Rich Text Editors
+  - [prosemirror](https://github.com/microsoft/FluidFramework/tree/main/examples/data-objects/prosemirror)
+  - [smde](https://github.com/microsoft/FluidFramework/tree/main/examples/data-objects/smde)
+  - [draft-js](https://github.com/microsoft/FluidExamples/tree/main/draft-js)
+
+- Plain Text Editor Implementations
+  - [collaborativeTextArea](https://github.com/microsoft/FluidFramework/blob/main/examples/data-objects/react-inputs/src/CollaborativeTextArea.tsx)
+  - [collaborativeInput](https://github.com/microsoft/FluidFramework/blob/main/examples/data-objects/react-inputs/src/collaborativeInput.tsx)
+
+## SharedObjectSequence and SharedNumberSequence
+
+SharedObjectSequence and SharedNumberSequence are very similar distributed data structures. The only difference is the
+type of content they support. SharedNumberSequence only supports numbers as content, while SharedObjectSequence supports
+any JSON serializable object. Both DDSes support inserting, removing, and annotating content. Each piece of content --
+that is, each number or object -- will occupy a single position in the sequence. The length of the sequence is the count
+of content items in the sequence.
+
+An important note is that, unlike an array, positions are not guaranteed remain constant. The position of an item can
+change as content is added or removed from the sequence. To track or pass a reference to a specific piece of content
```

Comment:
This looks like a really important technique that should get more emphasis and a concrete example.